g3web/src/struts.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE struts PUBLIC "-//Apache Software Foundation//DTD Struts  
+Configuration 2.1//EN" "http://struts.apache.org/dtds/struts-2.1.dtd">
+<struts>
+	<package name="o" extends="struts-default">

```

Comment:
@karthikvenkata @pranaychand  name="o" ? Everything should have a meaning to it. Please rename "o"